tools/samtools/examples/1.21.html

```diff
@@ -0,0 +1,20 @@
+<script src="https://biowasm.com/cdn/v3/aioli.js"></script>
+<script type="module">
+const CLI = await new Aioli(["samtools/1.21"]);
+
+// Show reads from toy.sam. Try replacing "-f 16" with "-f 0".
+const file = "/samtools/examples/toy.sam";
+const output = await CLI.exec(`samtools view -f 16 ${file}`);
+
+// Test out new command in v1.17
+const output2 = await CLI.exec(`samtools reset ${file}`);
```

Comment:
It would be useful to have an example that only works on 1.21 so we're sure it works correctly. Maybe `samtools fasta` with `--output` using `.gz`

Reply:
Sure! About bedtime on this side of the pool, so I'm not feeling to creative - added just a view of the created .gz. It may be that this actually would silently just give the same result on an earlier version, but at least it shows clearly if on inspects the files. :)